www/havm10/homework-05/js/main.js

```diff
@@ -0,0 +1,143 @@
+const pi = Math.PI;
+
+/*
+1)
+Pepe's age. Vypište na konzoli smysluplnou oznamovací větu ohledně věku Pepy, pokud znáte jeho rok narození. Použijte proměnné a pro výpis použijte zřetězení stringů. Jako názvy proměnných používejte anglické pojmy.
+*/
+
+var yearOfBirth = 1994;
+var thisYear = new Date().getFullYear();
+var age = thisYear - yearOfBirth;
+
+console.log('Pepovy je: ' + age + ' let');
+
+/*
+2)
+WTF (wow, that's fun). Vypište teplotu v Fahrenheiht, pokud znáte teplotu v Celsius, a také naopak. Formát výpisu je: 20°C =  68°F resp. 68°F = 20°C. Výpočet probíhá takto:
+    1) z C na F: vynásobit devíti, vydělit pěti a přičíst 32.
+    2) z F na C: odečíst 32, vynásobit pěti a vydělit devítkou.
+*/
+
+var celsium = 0;
+var fahrenheiht = 100;
+
+var celsiumToFahrenheiht = (celsium - 32) * 5 / 9;
```

Comment:
tohle moc neodpovida tomu vypoctu z C na F :)

Reply:
opraveno, původně jsem to dělal jako funkci a při překopírování jsem udělal chybu.